hermes/send-email.js

```diff
@@ -41,11 +44,21 @@ const sendEmail = (options: Options) => {
         TemplateModel: TemplateModel,
         Tag: Tag,
       },
-      err => {
+      async err => {
         if (err) {
-          console.log('Error sending email:');
-          console.log(err);
-          return rej(err);
+          console.error('Error sending email:');
+          console.error(err);
+
+          // 406 means the user became inactive, either by having an email
+          // hard bounce or they marked as as spam
+          if (err.code === 406) {
+            console.error(`Deactivating future notifications for ${To}`);
+            return await deactiveUserEmailNotifications(To)
+              .then(() => rej(err))
```

Comment:
Why are we `console.error`ing here if we know what the problem is anyway? This code should read:

```JS
async err => {
  if (err.code === 406) {
    return deactiveUserEmailNotifications(To)
      .catch(err => rej(err))
  }
  
  console.error(`Error sending email:`, err);
}
```

Sentry picks up `console.error`s, and we don't really want to get a log of an error we handled properly anyway, right?